src/main/java/com/rocketden/main/service/GameService.java

```diff
@@ -0,0 +1,49 @@
+package com.rocketden.main.service;
+
+import com.rocketden.main.controller.v1.BaseRestController;
+import com.rocketden.main.dao.RoomRepository;
+import com.rocketden.main.dto.game.StartGameRequest;
+import com.rocketden.main.dto.room.RoomDto;
+import com.rocketden.main.dto.room.RoomMapper;
+import com.rocketden.main.exception.RoomError;
+import com.rocketden.main.exception.UserError;
+import com.rocketden.main.exception.api.ApiException;
+import com.rocketden.main.model.Room;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.messaging.simp.SimpMessagingTemplate;
+import org.springframework.stereotype.Service;
+
+@Service
+public class GameService {
+
+	private static final String START_GAME_SOCKET_PATH = BaseRestController.BASE_SOCKET_URL + "/%s/start-game";
+
+	private final RoomRepository repository;
+	private final SimpMessagingTemplate template;
+
+	@Autowired
+	public GameService(RoomRepository repository, SimpMessagingTemplate template) {
+		this.repository = repository;
+		this.template = template;
+	}
+
+	// Send request to redirect users to game when host clicks start
+	public RoomDto startGame(StartGameRequest request) {
+		String socketPath = String.format(START_GAME_SOCKET_PATH, request.getRoomId());
+		template.convertAndSend(socketPath, request.getRoomId());
```

Comment:
I think this should be placed at the end after all the permission/room verification (otherwise this will send a socket message even if the POST request fails later down the line)